### ISSUE TITLE
Enable injecting test runner

### ISSUE THREAD

**Issue by la10736** [la10736/rstest#91]
We provide a way to define a test runner to use instead the default one (`test` or `async-std::test` for async function).

If the given attributes ends with `test` (the last segment in `Path`) we don't add `test` attribute to every test but just leave the provided one.

By this approach we can give a way to use `rstest` with a lot of other tools without write an ad hoc support.

As example follow code just works without care about what `actix_rt::test`. 

```rust
#[rstest(a, b, case (1, 2), case(3, 4))]
#[actix_rt::test]
async fn foo(a: u32, b: u32) {}
```

**Comment by la10736** [comment]
- [x] Impl
- [ ] Integration test
- [ ] Unit tests
- [ ] Changelog
- [ ] Docs

**Comment by DanielJoyce** [comment]
Does not work for #[rocket::async_test]

**Comment by la10736** [comment]
Unfortunately this case is not supported yet.
"Just the attributes that ends with test (last path segment) can be injected."
In this case last path segment is `async_test` and not just `test`.

I can try to think to something looser or add some custom rules... I've not a clear position about this :(

**Comment by la10736** [comment]
Should be fixed when we'll implement #228
